profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.module

```diff
@@ -343,7 +343,7 @@ function nexteuropa_multilingual_preprocess_field(&$variables) {
       $variables['label'] = i18n_field_translate_property($instance, 'label', $lang);
     }
     // Translate field value (for select list field).
-    if ($translated_value = i18n_field_translate_allowed_values($field, $lang)) {
+    if ($field['type'] !== 'taxonomy_term_reference' && $translated_value = i18n_field_translate_allowed_values($field, $lang)) {
```

Comment:
I think it would be better to limit field type to "select list" (instead of exclude taxonomy term):
```if ($field['type'] == 'list_text' && .....)```